net/vallumd/files/vallumd.conf

```diff
@@ -0,0 +1,5 @@
+config vallumd mqtt
+	option host 127.0.0.1
+	option port 1883
+	list topics blacklist4
+	list topics blacklist6
```

Comment:
Fine with me.  Though I'd like to note that netifd uses disabled as the name of this option and I guess many packages in this feed tend to do their own bool check without the help of config_get_bool ;)